File: src/targets/ruby/faraday/fixtures/application-form-encoded.rb
require 'faraday'

data = {
  :foo => "bar",
  :hello => "world",
}

conn = Faraday.new(
  url: 'http://mockbin.com',
  headers: {'Content-Type' => 'application/x-www-form-urlencoded'}
)

response = conn.post('/har') do |req|
  req.body = URI.encode_www_form(data)
end

puts response.status
puts response.body

File: src/targets/dart/http/fixtures/application-form-encoded.dart
import 'package:http/http.dart' as http;

void main() async {

  final headers = {
    'content-type': 'application/x-www-form-urlencoded',
  };

  final response = await http.post(
    Uri.parse('http://mockbin.com/har'),
    headers: headers,
    body: "foo=bar&hello=world",
  );

  print(response.statusCode);
  print(response.body);

}